jdaviz/components/plugin_add_results.vue

```diff
@@ -52,6 +52,17 @@
 
     <slot></slot>
 
+    <v-row v-if="auto_update_result !== undefined">
+      <v-switch
+        v-model="auto_update_result"
+        @change="(e) => {$emit('update:auto_update_result', auto_update_result)}"
+        label="Auto-update result"
+        hint="Regenerate the resulting data-product whenever any inputs are changed"
```

Comment:
can you change this message to be specific within in the plugin. for spectral extraction, say something like 'auto-update extracted spectrum when selected data is changed' ?

Reply:
This is a good idea - let's revisit this if the switch ends up staying visible (there is a chance when we swap out the glue implementation, that the switch won't even be available to the user, and it would take some work and thinking about the text to pass the property through to the re-usable component).